### PR TITLE
improved the way of extracting importnames from code

### PR DESCRIPTION
**Pull request by prudhvije**

### CODE REVIEW

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/RefreshAction.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ public final void runAction(final AnActionEvent anActionEvent) throws IOExceptio
             Set<String> imports = editorDocOps.getImports(projectEditor.getDocument());
             Set<String> lines = editorDocOps.getLines(projectEditor, windowObjects.getDistance());
             Set<String> internalImports = editorDocOps.getInternalImports(project);
-            Set<String> externalImports = 
+            Set<String> externalImports =
```
Comment (prudhvije) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/RefreshAction.java]: Checkstyle reports that there is extra trailing space for this line. So removed that.

**Comment by ScrapCodes** [comment]
I hope we can add tests for this ?

**Comment by prudhvije** [comment]
Yes.. #93 PR is tests this only. Will open it again.

**Comment by prudhvije** [comment]
Updated through #98 because of merge conflicts.

**Pull request closed without merging**